### PR TITLE
Update copy around on Delete Org modal

### PR DESCRIPTION
**Pull request by selfcontained**
## Description

Updates copy in the delete org modal.

## Related Issue(s)

Fixes WEB-368

## How to test

Open the delete org modal and verify the copy is updated.

#### Preview status

gitpod:summary

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

File: components/dashboard/src/teams/TeamSettings.tsx [hunk 1]
```diff
@@ -170,6 +170,7 @@ export default function TeamSettingsPage() {
                         All <b>members</b> of this organization will lose access to this organization, associated
                         projects and workspaces.
                     </li>
+                    <li className="ml-5">Any free credit allowances granted to this organization will be lost.</li>
```
Comment (gtsiolis) [inline components/dashboard/src/teams/TeamSettings.tsx:173]: **nitpick**: Minor copy suggestion—feel free to pick up parts of it or leave it as is.

```suggestion
                    <li className="ml-5"Any remaining <b>credits</b> will be lost.</li>
```

**Review by gtsiolis — APPROVED** [review]
UX LGTM. 🏁 

Thanks, @selfcontained. 🌮 🌮

File: components/dashboard/src/teams/TeamSettings.tsx (hunk 1 above)
Comment (selfcontained) [reply components/dashboard/src/teams/TeamSettings.tsx:173]: I think the wording of any free credit allowances is a bit more explicit and intentional in describing that you'll lose your monthly free credit allowance if this org has one (i.e. the first org upon joining).

Comment (gtsiolis) [reply components/dashboard/src/teams/TeamSettings.tsx:173]: Fair points, @selfcontained. However, the credits we may grant to organizations may not be given for free but I trust you as a natural language speaker here. 🙇

**Pull request merged**